src/components/list/list.scss

```diff
@@ -16,63 +17,161 @@ $item-padding-top: 0px !default;
 $item-padding-right: 0px !default;
 $item-padding-left: 0px !default;
 $item-padding-bottom: 0px !default;
+$list-item-padding-vertical: 0px !default;
+$list-item-padding-horizontal: $baseline-grid * 2 !default;
+$list-item-primary-width: $baseline-grid * 7 !default;
+$list-item-primary-avatar-width: $baseline-grid * 5 !default;
+$list-item-primary-icon-width: $baseline-grid * 3 !default;
+$list-item-secondary-left-margin: $baseline-grid * 2 !default;
+$list-item-text-padding-top: $baseline-grid * 2 !default;
+$list-item-text-padding-bottom: $baseline-grid * 2.5 !default;
 
 md-list {
+  display: block;
   padding: $list-padding-top $list-padding-right $list-padding-bottom $list-padding-left;
+
+  .md-subheader {
+    font-weight: 500;
+    font-size: 0.875em;
+  }
 }
 
 md-item {
-
+  &.md-no-style,
+  .md-no-style {
+    padding: $list-item-padding-vertical $list-item-padding-horizontal;
+    flex: 1;
+  }
+  &.md-with-secondary {
+    padding-right: $list-item-padding-horizontal;
+  }
+  &.md-clickable:hover {
+    cursor: pointer;
+  }
 }
 
-md-item-content {
+md-item, md-item > .md-no-style {
   display: flex;
+  justify-content: flex-start;
   align-items: center;
-  flex-direction: row;
-  box-sizing: border-box;
 
-  position: relative;
+  // Layout for controls in primary or secondary divs, or auto-infered first child
+  & > div.md-primary > md-icon,
+  & > div.md-secondary > md-icon,
+  & > md-icon:first-child, 
+  > md-icon.md-secondary {
+    width: $list-item-primary-icon-width;
+    margin-top: 12px;
+    margin-bottom: 12px;
+    box-sizing: content-box;
+    &:focus {
+      padding: 6px;
```

Comment:
So that icon buttons look good. If you tab through some of the secondary controls (icons with `ng-click`) you can see how they get applied.